src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.4.0-rc.2
+
```

Comment:
Released date is needed. See previous sections.

Reply:
Gotcha, fixed.